6_visualize/src/prep_storm_line_fun.R

```diff
@@ -3,7 +3,7 @@ prep_storm_line_fun <- function(storm_line_ind_file){
   storm_line_sf <- readRDS(sc_retrieve(storm_line_ind_file))
 
   plot_fun <- function(){
-    plot(storm_line_sf, add = TRUE, col = 'black', lwd = 2)
+    plot(storm_line_sf[,1], add = TRUE, col = 'black', lwd = 2)
```

Comment:
👍

Reply:
This does fix the error, but note that we can use the `attr(storm_line_sf, "sf_column")` attribute to get the geometry column

Reply:
:+1:

Reply:
https://github.com/USGS-VIZLAB/vizstorm-GIF/pull/61